x/evm/wrappers/bank.go

```diff
@@ -0,0 +1,111 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package wrappers
+
+import (
+	"context"
+	"math/big"
+
+	"cosmossdk.io/errors"
+	sdkmath "cosmossdk.io/math"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/evmos/evmos/v20/x/evm/config"
+	"github.com/evmos/evmos/v20/x/evm/types"
+)
+
+var _ types.BankWrapper = BankWrapper{}
+
+// BankWrapper is a wrapper around the Cosmos SDK bank keeper
+// that is used to manage an evm denom with a custom decimal representation.
+type BankWrapper struct {
+	types.BankKeeper
+}
+
+// NewBankWrapper creates a new BankWrapper instance.
+func NewBankWrapper(
+	bk types.BankKeeper,
+) *BankWrapper {
+	return &BankWrapper{
+		bk,
+	}
+}
+
+// ------------------------------------------------------------------------------------------
+// Bank wrapper own methods
+// ------------------------------------------------------------------------------------------
+
+// MintAmountToAccount converts the given amount into the evm coin scaling
+// the amount to the original decimals, then mints that amount to the provided account.
+func (w BankWrapper) MintAmountToAccount(ctx context.Context, recipientAddr sdk.AccAddress, amt *big.Int) error {
+	coin := sdk.NewCoin(config.GetEVMCoinDenom(), sdkmath.NewIntFromBigInt(amt))
+
+	convertedCoin, err := convertEvmCoinFrom18Decimals(coin)
+	if err != nil {
+		return errors.Wrap(err, "failed to mint coin to account in bank wrapper")
+	}
+
+	coinsToMint := sdk.NewCoins(convertedCoin)
+	if err := w.MintCoins(ctx, types.ModuleName, coinsToMint); err != nil {
```

Comment:
use BankKeeper method
```suggestion
	if err := w.BankKeeper.MintCoins(ctx, types.ModuleName, coinsToMint); err != nil {
```

Reply:
It is already the bank keeper method